src/guessTheNumber.js

```diff
@@ -0,0 +1,16 @@
+const randomNum = Math.floor(Math.random() * 100)
+export const guessTheNumber = (input) => {
+  if(Number.isInteger(input) && input < randomNum) {
+    return 'Your guess is kinda low'
+  } else if(Number.isInteger(input) && input > randomNum) {
+    return 'You too high!!!'
+  } else if(input === randomNum) {
+    return 'Yeee you win!!!'
+    process.exit(0)
+  } else if(input.toLowerCase() === 'exit'){
+    process.stdout.write('game over! way to quit')
+    process.exit(0)
+  } else if(!input || isNaN(input)) {
+    return 'Insert only integers RAWR!!!'
```

Comment:
I think you could have wrapped the preceding if statements inside of this with an additional check for Number.isInteger(input). This would have avoided duplicate code and left the if-statements a little more readable.